docs/using-scylla/cassandra-compatibility.rst

```diff
@@ -36,8 +36,11 @@ Interfaces
      - Compatible with Cassandra 2.1
      - 
    * - SSTable format (all versions)
-     - 3.11(mc / md ), 2.2(la), 2.1.8 (ka)
-     - ``md`` format is supported in ScyllaDB Open Source 4.3 and later, and ScyllaDB Enterprise 2021.1.0 and later
+     - 3.11(mc / md / me), 2.2(la), 2.1.8 (ka)
+     - | ``me`` - supported in ScyllaDB Open Source 5.1 and ScyllaDB Enterprise 2022.1.0 (and later)
```

Comment:
this should be 2022.2.0

(2022.1.0 is based on 5.0 that doesn't support the ME format yet)

Reply:
@bhalevy Thanks, I've replaced it with 2022.2.0.